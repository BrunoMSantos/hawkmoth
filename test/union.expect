.. c:type:: union foo

   
   Union documentation.
   
   

.. c:member:: int foo.foo

   
   int member 1.
   

.. c:member:: int foo.bar

   
   int member 2.
   

.. c:member:: struct _baz foo._baz

   
   struct member.
   

