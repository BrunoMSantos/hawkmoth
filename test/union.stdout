
.. c:type:: union foo

   Union documentation.


   .. c:member:: int foo

      int member 1.


   .. c:member:: int bar

      int member 2.


   .. c:member:: struct _baz _baz

      struct member.

